frontend/include/prelude.html

```diff
@@ -1,15 +1,21 @@
 <!-- Primary Meta Tags -->
 <meta name="title" content="ETHSBell" />
-<meta name="description" content="ETHSBell is designed to help Evanston Township High School students be more familiar with the bell schedule and navigate their school day." />
+<meta
+	name="description"
+	content="ETHSBell is designed to help Evanston Township High School students be more familiar with the bell schedule and navigate their school day."
```

Comment:
This description is a bit long but you don't need to change it

Reply:
This fits within the [recommended length](https://www.google.com/search?q=meta+description+recommended+length) that multiple sites give.